content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -714,17 +717,33 @@ serves the following additional paths at its HTTP(S) ports.
   The output is similar to this:
 
   ```none
-  PriorityLevelName, Index,  PendingRequests, ExecutingRequests, VirtualStart,
-  workload-high,     0,      0,               0,                 0.0000,
-  workload-high,     1,      0,               0,                 0.0000,
-  workload-high,     2,      0,               0,                 0.0000,
+  PriorityLevelName, Index,  PendingRequests, ExecutingRequests, SeatsInUse, NextDispatchR,   InitialSeatsSum, MaxSeatsSum, TotalWorkSum
+  workload-low,      14,     27,              0,                 0,          77.64342019ss,   270,             270,         0.81000000ss
+  workload-low,      74,     26,              0,                 0,          76.95387841ss,   260,             260,         0.78000000ss
+  ...
+  leader-election,   0,      0,               0,                 0,          5088.87053833ss, 0,               0,           0.00000000ss
+  leader-election,   1,      0,               0,                 0,          0.00000000ss,    0,               0,           0.00000000ss
   ...
-  leader-election,   14,     0,               0,                 0.0000,
-  leader-election,   15,     0,               0,                 0.0000,
+  workload-high,     0,      0,               0,                 0,          0.00000000ss,    0,               0,           0.00000000ss
+  workload-high,     1,      0,               0,                 0,          1119.44936475ss, 0,               0,           0.00000000ss
   ```
 
+  Explanation for selected column names:
+  - `NextDispatchR`: The R progress meter reading, in units of seat-seconds, at
```

Comment:
Who cares about the meaning of `R`?

Reply:
K8s users should not care.
But the column name has `R`. Saying `The progress meter reading` may mislead the reader think `R` means `Reading`.
I can't find anything better. @tengqm Any suggestions?

Reply:
Right. If K8s users **mostly** won't care about this debug API, maybe we should consider moving this content to the references section. It is not some basic concepts users should know anyway.

Reply:
Most k8s users should not care about this, but some cluster operators may care. When they are reading the `debug endpoint` section, it means they will want to understand what each column means here.
I'd suggest keep the column explanation inline with the `debug endpoint` section unless we want to move `debug endpoint` section to somewhere else.

Reply:
There is existing documentation around cluster troubleshooting.

We might want to shift that explanation to be within https://kubernetes.io/docs/tasks/debug/debug-cluster/, and change the APF docs to link to one section of those troubleshooting docs.

Reply:
If we want to move the troubleshooting related content, we can create a new doc for APF under https://kubernetes.io/docs/tasks/debug/debug-cluster/ and we will move https://v1-23.docs.kubernetes.io/docs/concepts/cluster-administration/flow-control/#diagnostics and https://v1-23.docs.kubernetes.io/docs/concepts/cluster-administration/flow-control/#debug-endpoints there while we keep metrics section as is.
If this make sense, I can make the change.